content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -5,24 +5,37 @@ title: Choosing a Jenkins version to build against
 
 You need to balance compatibility and features:
 
-* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin. 
+* *Keeping the Jenkins version your plugin builds against low* will allow more users to install and use your plugin.
 In particular, the LTS Release Line is based on slightly older releases to provide a more stable experience for conservative users.
-* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin. 
-You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases. 
+* *Building against recent Jenkins versions* allows you to use recently added core features and API from your plugin.
+You will also use that newer version for `mvn hpi:run`, so it may be easier to test your plugin with newer Jenkins releases.
 Additionally, since features are sometimes moved from core into plugins, depending on a more recent Jenkins version will make your plugin's dependencies on what used to be core features explicit, otherwise your plugin will not work.
-* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (2020-03) this is 2.164.x.
+* *Use at least the minimum version supported by the update center*, the supported update center versions can be found at link:https://updates.jenkins.io[updates.jenkins.io], at the time of writing (August 2020) this is 2.176.x.
 
-There are link:https://stats.jenkins.io/pluginversions/[statistics of core versions by plugin] that can help you decide.
+There are link:https://stats.jenkins.io/pluginversions/[statistics of core versions by plugin] that can help you decide. These are updated monthly.
 
-These are updated monthly.
+
+== Suggested minimum version
 
 General tip: When upgrading, choose the link:/changelog-stable/[newest LTS version] that doesn't alienate the majority of users of your latest few releases (by requiring a newer Jenkins than they have).
 
-## Changing the minimum required version
+If you do not have special requirements then a good idea is to support the latest three
+LTS minor versions. This helps to ensure that teams that are slower at adapting new Jenkins versions 
+will not be forced to update their Jenkins too frequently. Additionally, other plugins that 
+depend on your plugin also need to upgrade their minimum Jenkins version to your Jenkins version, 
+otherwise the injected tests will fail.
+
+Example: if we are currently at Jenkins 2.248 then the latest LTS version is 
+2.235.x. Then it would make sense to also support 2.222.x and 2.204.x. When setting the Jenkins version
+please also make sure to use the .1 patch version of the LTS version: 2.204.1, or 2.222.1. Using
+a newer patch version like 2.222.6 causes  build problems in depending plugins that depend on the
```

Comment:
This explanation makes no sense to me, I responded in the mailing list thread.

Reply:
Here are some problems that are caused by plugins updating their baseline. My plugin had a baseline of 2.204.4, the same baseline was used in my ATH tests. It depends directly or indirectly on two API plugins: snakeyaml-api-plugin and jackson2-api-plugin. These plugins have been recently updated to 2.204.6 and 2.222.4, respectively. This caused failures for the maven-enforcer, the generated injected integration tests, and the UI tests for my plugins. And not because of a bug - just because of an inconsistent Jenkins baseline selection.

Reply:
I do not get it. If your baseline is 2.204.x then just do not pick newer versions of these plugins as `test`-scoped dependencies, unless and until you update your own baseline. Of course if you use the `bom` then you do not need to even think about this issue for covered dependencies.